Sources/Sentry/SentryTransportFactory.m

```diff
@@ -34,9 +35,12 @@ @implementation SentryTransportFactory
         SentryRateLimitParser *rateLimitParser = [[SentryRateLimitParser alloc] init];
         id<SentryRateLimits> rateLimits = [[SentryDefaultRateLimits alloc] initWithRetryAfterHeaderParser:retryAfterHeaderParser andRateLimitParser:rateLimitParser];
         
+        SentryEnvelopeRateLimit * envelopeRateLimit = [[SentryEnvelopeRateLimit alloc] initWithRateLimits:rateLimits];
+        
         return [[SentryHttpTransport alloc] initWithOptions:options
                                           sentryFileManager:sentryFileManager sentryRequestManager: requestManager
-                                           sentryRateLimits: rateLimits];
+                                           sentryRateLimits: rateLimits
+                                    sentryEnvelopeRateLimit: envelopeRateLimit];
```

Comment:
```suggestion
                                           sentryEnvelopeRateLimit: envelopeRateLimit];
```

Reply:
We align here by the `:` so I believe the code in question is rightly formatted

Reply:
I did from my phone, maybe that's why, they were not aligned, but it is now from browser